cluster-autoscaler/utils.go

```diff
@@ -28,11 +28,40 @@ import (
 	apiv1 "k8s.io/kubernetes/pkg/api/v1"
 	metav1 "k8s.io/kubernetes/pkg/apis/meta/v1"
 	kube_client "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_5"
+	kube_record "k8s.io/kubernetes/pkg/client/record"
 	"k8s.io/kubernetes/plugin/pkg/scheduler/schedulercache"
 
 	"github.com/golang/glog"
 )
 
+// AutoscalingContext contains more-or-less constant, configuration-related objects passed to
```

Comment:
Could you please fix the description of AutoscalingContext? It sounds vague right now.